gems/aws-sdk-dynamodb/lib/aws-sdk-dynamodb/plugins/extended_retries.rb

```diff
@@ -4,25 +4,50 @@ module Aws
   module DynamoDB
     module Plugins
       class ExtendedRetries < Seahorse::Client::Plugin
+        DEFAULT_BACKOFF = lambda do |c|
+          return if c.retries < 1
 
-        option(:retry_limit,
+          delay = 2**(c.retries - 1) * c.config.retry_base_delay
+          if (c.config.retry_max_delay || 0) > 0
+            delay = [delay, c.config.retry_max_delay].min
+          end
+          jitter = c.config.retry_jitter
+          jitter = JITTERS[jitter] if jitter.is_a?(Symbol)
```

Comment:
Will this work? I don't think `JITTERS` is defined here

Reply:
Ah good catch. It looks like this is not really tested either.